docs/EnterpriseScaleAnalytics-GitHubActionsDeployment.md

```diff
@@ -68,6 +68,7 @@ To begin, please open the [infra/params.dev.json](/infra/params.dev.json). In th
 | tags | Specifies the tags that you want to apply to all resources. | {`key`: `value`} |
 | administratorPassword | Specifies the administrator password of the sql servers. Will be automatically set in the workflow. **Leave this value as is.** | `<your-secure-password>` |
 | synapseDefaultStorageAccountFileSystemId | Specifies the resource ID of the default storage account file system for synapse. | `/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Storage/storageAccounts/{storage-name}/blobServices/default/containers/{container-name}` |
+| enableDataExplorerPool | Specifies whether an Azure Data Explorer Pool should be deployed inside your Synapse workspace as part of the template. If you selected dataFactory as processingService, leave this value as is. | `true` or `false` |
```

Comment:
Should "dataFactory" be changed to `dataFactory`.

Since we are referring to an attribute value for "processingService", should we change it to `processingService`  (add backticks)?

Thanks

Reply:
Let's get this done as part of another PR.